src/components/Thread.vue

```diff
@@ -94,8 +122,43 @@ export default {
 	},
 	created() {
 		this.resetThread()
+		window.addEventListener('resize', debounce(500, this.updateParticipantsToDisplay))
+	},
+	beforeDestroy() {
+		window.removeEventListener('resize', debounce(500, this.updateParticipantsToDisplay))
```

Comment:
this doesn't actually work because every time you call `debounce(500, this.updateParticipantsToDisplay)` you will get a *new function*. See what we did with `saveDraft` and `saveDraftDebounced` in `Composer.vue`. Hope this makes it more clear.

Reply:
should be good now